lib/DiFactory.js

```diff
@@ -132,7 +132,7 @@ DiFactory.prototype._getDefaultSize = function(element) {
     return { width: 50, height: 50 };
   }
 
-  if (is(elementType, 'bpmn:StartEvent') || is(elementType, 'bpmn:EndEvent')) {
+  if (is(elementType, 'bpmn:StartEvent') || is(elementType, 'bpmn:EndEvent') || is(elementType, 'bpmn:IntermediateCatchEvent')) {
```

Comment:
We should use a general type here:
```suggestion
  if (is(elementType, 'bpmn:Event'))
```
Notice that the implementation still misses both boundary and throw events.